src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -23,15 +23,77 @@
 	<NcAppNavigation :aria-label="t('spreed', 'Conversation list')">
 		<div class="new-conversation"
 			:class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
-			<SearchBox v-model="searchText"
+			<SearchBox ref="searchbox"
+				v-model="searchText"
 				class="conversations-search"
+				:class="{'conversations-search--expanded': isFocused,
+					'conversations-search--filter' : isFiltered }"
 				:is-searching="isSearching"
+				@blur="setIsFocused(false)"
+				@focus="setIsFocused(true)"
 				@input="debounceFetchSearchResults"
 				@submit="onInputEnter"
 				@keydown.enter.native="handleEnter"
 				@abort-search="abortSearch" />
-			<NewGroupConversation v-if="canStartConversations" />
+
+			<!-- Options -->
+			<div class="options"
+				:class="{'options--hidden': isFocused}">
+				<NcActions ref="filterMainBtn"
+					class="filters-button">
+					<template #icon>
+						<FilterIcon v-if="!isFiltered" :size="15" />
+						<FilterCheckIcon v-else-if="isFiltered" :size="15" />
+					</template>
+					<NcActionButton close-after-click
+						class="filterButton--Option"
+						@click="handleFilter('is:mentioned')">
+						<template #icon>
+							<AtIcon :size="20" />
+						</template>
+						{{ t('spreed','Filter unread mentions') }}
+					</NcActionButton>
+
+					<NcActionButton close-after-click
+						class="filterButton--Option"
+						@click="handleFilter('is:unread')">
+						<template #icon>
+							<MessageBadge :size="20" />
+						</template>
+						{{ t('spreed','Filter unread messages') }}
+					</NcActionButton>
+
+					<NcActionButton v-if="isFiltered"
+						close-after-click
+						class="filterButton--Clear"
+						@click="clearFilter">
+						<template #icon>
+							<FilterRemoveIcon :size="20" />
+						</template>
+						{{ t('spreed', 'Clear filters: {filterText}', { filterText: filterText }) }}
```

Comment:
I think, Joas meant just 'Clear (all) filters'. We can't translate this message as it's divided in two parts, before and after colon. It's different for some languages, and translators couldn't rewrite it to sound correctly